docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -1,34 +1,57 @@
-==========================
-File system access control
-==========================
+=========================
+File-based access control
+=========================
 
-This access control plugin allows you to specify authorization rules in a JSON file.
+To secure access to data in your cluster, you can implement file-based access
+control where access to data and operations is defined by rules declared in
+manually-configured JSON files.
+
+There are two types of file-based access control:
+
+* **System-level access control** uses the access control plugin with a single
+  JSON file that specifies authorization rules for the whole cluster.
+* **Catalog-level access control** uses individual JSON files for each catalog
+  for granular control over the data in that catalog, including column-level
+  authorization.
+
+File-based access control process from the top to bottom of the JSON file with
+an implicit deny. If a rule matches the user then the corresponding operation
+is allowed. If no rules match, the operation is denied.
```

Comment:
```suggestion
File-based access control rules are processed in order from top to bottom of the JSON rules file. The first matching rule is applied. If no rules match, the operation is denied.
```

Note that a match doesn't have to mean "allow". e.g. `"owner": false` in a rule will remove ownership for the matched user instead of granting it.

The concept is that there is a list of "rules" (i.e. what to do - could be allow/deny/whatever) and the user information is matched against the list of rules. The first rule that matches is executed. If none match then the permission is denied.

(Feel free to use this info to reword/adjust my suggestion above).

Reply:
Removed this paragraph per your comment below, since it's described in the respective intros for system-level and catalog-level.